.gitignore

```diff
@@ -21,6 +21,7 @@ update.json
 !/.github
 !/.gitignore
 !*.rpdk-config
+!.mockery.yaml
```

Comment:
what / why are we ignoring here?

Reply:
there is a statement just above that ignores all hidden files:
```
# any "dot"-file, anywhere almost
**/.*
```
this allows the mockery file to be included

Reply:
ah, ok, thx